### PR TITLE
Fix local path being passed to Verbnet class during instantiation 

### PR DESCRIPTION
**Pull request by sbhaktha**
so that it gets serialized without local paths.

@jkinkead : Jesse, changes as discussed!

### CODE REVIEW

File: tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/ml/Verbnet.scala [hunk 1]
```diff
@@ -14,11 +16,16 @@ import DefaultJsonProtocol._
   * (http://projects.csail.mit.edu/jverbnet/),  to quickly look up various vernbet
   * features for a verb.
   */
-case class Verbnet(verbnetPath: String) {
+case class Verbnet(groupName: String, artifactName: String, version: Int) {
 
   // Construct the index and open it
   @transient val index = {
-    val url = new URL("file", null, verbnetPath)
+    val verbnetPath: java.nio.file.Path = Datastore.directoryPath(
```
Comment (jkinkead) [inline tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/ml/Verbnet.scala]: I belief there's a call that can get a file URL directly.

Also, you should leave off the type declaration - it's just for documenting the return type in Dirk's example code.

**Comment by jkinkead** [comment]
One comment!

**Comment by sbhaktha** [comment]
`directoryURL` is the Datastore API that returns a URL directly, however it just returns the datastore link -- as a zip file. That's not what we want here. Spoke to Dirk to make sure. But he actually commented on something else-- about creating `URL` from `Path`. Changed that because the java APIs might be more robustly handling various possibly malformatted urls.

File: tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/ml/Verbnet.scala [hunk 2]
```diff
@@ -14,11 +16,16 @@ import DefaultJsonProtocol._
...
 
   // Construct the index and open it
   @transient val index = {
-    val url = new URL("file", null, verbnetPath)
+    val verbnetPath = Datastore.directoryPath(
+      groupName,
+      artifactName,
+      version
+    )
+    val url = verbnetPath.toUri.toURL
```
Comment (jkinkead) [inline tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/ml/Verbnet.scala]: You could inline below, if you like.

**Comment by jkinkead** [comment]
LGTM! I had a small comment, but feel free to merge when you're happy!

**Pull request merged**